src/components/Combobox/components/index.js

```diff
@@ -0,0 +1,13 @@
+import ComboboxCore from './ComboboxCore';
+import ComboboxDropdown from './ComboboxDropdown';
+import ComboboxDropdownLists from './ComboboxDropdownLists';
+import ComboboxListbox from './ComboboxListbox';
+
+export * from './DropdownItems';
+
+export {
+  ComboboxCore,
```

Comment:
do we need/want to export core version?

Reply:
Thought about it and you raise a valid point, we should not export it. We should also not export `ComboboxLists` and `ComboboxDropdown`. I'll update the PR